macros for lpc_18xx43xx_emac_config
===================================================================
diff -urN lwip/examples/blinking_lwip/inc/lpc_18xx43xx_emac_config.h lwip-ciaa/examples/blinking_lwip/inc/lpc_18xx43xx_emac_config.h
--- lwip/examples/blinking_lwip/inc/lpc_18xx43xx_emac_config.h	2014-12-04 16:51:24.000000000 -0300
+++ lwip-ciaa/examples_ciaa/blinking/inc/lpc_18xx43xx_emac_config.h	2016-06-28 13:55:28.313520407 -0300
@@ -65,6 +65,10 @@
 /* Array of slow memory address ranges for LPC_CHECK_SLOWMEM */
 #define LPC_SLOWMEM_ARRAY
 
+/* Build for RMII interface */
+#define USE_RMII
+#define BOARD_ENET_PHY_ADDR	0x01
+
 #ifdef __cplusplus
 }
 #endif
